helix-view/src/graphics.rs

```diff
@@ -250,14 +250,14 @@ impl From<Color> for crossterm::style::Color {
             Color::Blue => CColor::DarkBlue,
             Color::Magenta => CColor::DarkMagenta,
             Color::Cyan => CColor::DarkCyan,
-            Color::Gray => CColor::Grey,
-            Color::DarkGray => CColor::DarkGrey,
+            Color::Gray => CColor::DarkGrey,

```

Comment:
Any reason not to follow crossterm color names? I admit that those are still non-standard (it's usually black not gray, and the normal variants are the dark ones with a "bright" version)

Reply:
I think I answered my own question, you followed ANSI conventions :)